### PR TITLE
Add an insecure option to the openshift template

### PR DESCRIPTION
**Pull request by rthallisey**
Running the broker in insecure mode from the template should
only require changing a few settings.

Changes proposed in this pull request
 - Option to run the broker as insecure from the openshift template

fixes https://github.com/openshift/ansible-service-broker/issues/333

### CODE REVIEW

**Comment by rthallisey** [comment]
@shawn-hurley ya I was really back and forth on one or two flags.  The best solution is a section somewhere in our quickstart/developer guide explaining the exact steps, which still needs to be written.

**Comment by shawn-hurley** [comment]
does this sould right?
INSECURE -> false, TERMINATION -> reencrypt -> :white_check_mark: 
INSECURE -> true, TEMINATION -> reencrypt -> :red_circle: 
INSECURE -> false, TERMINATION -> edge -> :red_circle: 
INSECURE -> true, TEMINATION -> edge -> :white_check_mark: 
INSECURE -> false, TERMINATION -> passthrough -> should test this 
INSECURE -> true, TEMINATION -> passthrough -> should test this

File: templates/deploy-ansible-service-broker.template.yaml [hunk 1]
```diff
@@ -104,6 +104,8 @@ objects:
           env:
           - name: BROKER_CONFIG
             value: ${BROKER_CONFIG}
+          - name: INSCURE
```
Comment (shawn-hurley) [inline templates/deploy-ansible-service-broker.template.yaml]: TYPO: INSCURE -> INSECURE

Comment (shawn-hurley) [inline templates/deploy-ansible-service-broker.template.yaml]: Just noticed it sorry

**Comment by rthallisey** [comment]
@shawn-hurley 

INSECURE -> false, TERMINATION -> reencrypt -> :white_check_mark: 
INSECURE -> true, TEMINATION -> reencrypt -> :red_circle: 
INSECURE -> false, TERMINATION -> edge -> :red_circle: 
INSECURE -> true, TEMINATION -> edge -> :red_circle: 
INSECURE -> false, TERMINATION -> passthrough -> :white_check_mark:
INSECURE -> true, TEMINATION -> passthrough -> :white_check_mark:

**Comment by shawn-hurley** [comment]
@rthallisey I am pretty sure that  ```INSECURE -> true``` and ```TERMINATION -> edge``` will work. 

I also was unable to get ```TERMINATION -> passthrough``` working with ```INSECURE -> true```

**Comment by rthallisey** [comment]
The testing I did above was with the broker running in the cluster. I think the results are different running out of the cluster.

**Comment by eriknelson** [comment]
photo_2017-08-09_12-27-19

**Pull request merged**